### PR TITLE
sakhalin.info

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by J0WI** [comment]
sakh.com seems related, can you check that?

File: src/chrome/content/rules/sakhalin.info.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<!--9may. refused-->
```
Comment (J0WI) [inline src/chrome/content/rules/sakhalin.info.xml]: same on 70let.

**Comment by galeksandrp** [comment]
Added.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**